### ISSUE TITLE
Typescipt Compile issues

### ISSUE THREAD

**Issue by seanmccay** [dmtrKovalenko/material-ui-pickers#964]
![DERP](https://i.kym-cdn.com/photos/images/original/000/270/676/83b.png)

## Environment

| Tech                | Version |
|---------------------|---------|
| material-ui-pickers |     ^2.2.4    |
| material-ui         |    ^3.9.2     |
| React               |    ^16.8.5     |
| Browser             |    Chrome     |
| Peer library        |    "moment": "^2.22.2"     | 

| other pckgs   | Version |
|-----------------|--------|
| @types/react | ^16.8.8 |
| typescript | ^3.3.4000 |

## Steps to reproduce
1. install dependencies
2. run `tsc -p tsconfig.test.json --noEmit`
3. see compile issues

## Expected behavior
my project to compile

## Actual behavior
errors from material-ui-pickers during run of tsc.
Getting errors from: 
- InlineWrapper.d.ts
- DateTextField.d.ts
- ModalWrapper.d.ts

example: 
```
node_modules/material-ui-pickers/wrappers/InlineWrapper.d.ts:354:2018 - error TS2694: Namespace '"@material-ui/core/TextField"' has no exported
member 'TextFieldProps'.
```

## Live example
Not sure how to run a terminal command in codesandbox, but here's mine with the compile script: [CodeSandbox](https://codesandbox.io/s/ovqlo3jwxq?fontsize=14)

I'm assuming there's an issue with what version of @types I'm using or something along those lines, but I'm not sure what I should be on.

**Comment by seanmccay** [comment]
Hrmmm. Ok. I just realized that I had created a typings class and done:
```
declare module "@material-ui/core/Button";
declare module "@material-ui-core/Dialog";
etc...
```
because VSCode is telling me that it can't find the modules/typings for them. I removed those module declarations and everything compiled correctly. Not sure why VSCode is thinking it can't find the typings included in Material UI. So it was nothing to do with `material-ui-pickers` at all. 

Thanks!
